users/vnmm/vnmm.c

```diff
@@ -0,0 +1,23 @@
+/*
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef RGB_MATRIX_ENABLE
+#include "rgb_matrix_user.h"
+
+void matrix_init_user(void) {
+    rgb_matrix_init_user();
+}
+#endif
```

Comment:
1. You already have the `#ifdef` to conditionally `#include` at your `.h` file
1. You may want to do other stuff on the init later on (even if RGB is disabled)
1. Dont use `matrix_init` (meant to set things up so your MCU can scan the keys)
```suggestion
#include "vnmm.h"

void keyboard_post_init_user(void) {
#ifdef RGB_MATRIX_ENABLE
    rgb_matrix_init_user();
#endif
}
```